### ISSUE TITLE
Toggle Panorama hotkey on mac does not work

### ISSUE THREAD

**Issue by ebencollins** [nyordanov/panorama#1]
I see that the hotkey is set to "Ctrl+E" to toggle the panorama view. This translates to "Cmd+E" on MacOS, which doesn't trigger anything.

With legacy Tab Groups, I could use Cmd+Shift+E to toggle tab groups.

I added a mac keybind to manifest to make it work:
```
		"open-panorama": {
			"suggested_key": {
				"default": "Ctrl+E",
                "mac": "Ctrl+Shift+E"
			},
			"description": "Open panorama"
		}
```

**Comment by nyordanov** [comment]
Does `Ctrl+Shift+E` work for you at all? I initially tried to use that as the default shortcut, but it turned out that it is captured by the developer tools (it opens the "Network" tab).

**Comment by ebencollins** [comment]
`Ctrl+Shift+E` works for me on mac, but not on windows. `Ctrl+E` wasn't working on either with the latest version of Firefox Developer Edition. (Nothing happened on Mac, and it selected the URL bar on windows).

`Ctrl+Shift+E` does open dev tools/network on windows for me, but not Mac.

**Comment by nyordanov** [comment]
Fixed in v0.0.9
